clients/unshaded/src/main/java/tachyon/client/file/options/MkdirOptions.java

```diff
@@ -44,6 +50,15 @@ public Builder setRecursive(boolean recursive) {
     }
 
     /**
+     * @param underStorageType the under storage type to use
+     * @return the builder
+     */
+    public Builder setUnderStorageType(UnderStorageType underStorageType) {
+      mUnderStorageType = underStorageType;
+      return this;
+    }
```

Comment:
That is true, but that is simply what the current implementation does. Using UnderStorageType for the argument type (as opposed to boolean) allows us to implement support for ASYNC_PERSIST in the future, without changing the CreateOptions / MkdirOptions / OutStreamOptions API.